scipy/optimize/tests/test__remove_redundancy.py

```diff
@@ -141,12 +156,8 @@ def test_dense1(self):
         A[4, 2:] = 0
         b = np.zeros(A.shape[0])
 
-        A2 = A[[0, 1, 3, 4], :]
-        b2 = np.zeros(4)
-
         A1, b1, status, message = self.rr(A, b)
-        assert_allclose(A1, A2)
-        assert_allclose(b1, b2)
+        assert_(redundancy_removed(A1, A))
```

Comment:
Adding the old-school `assert_` is probably not desirable anymore now that `pytest` allows us to use `assert` and sometimes even provides better feedback from its "overload" of that functionality